javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,20 @@
+var fullName = 'Morris Wang';
+console.log('full name:', fullName);
+console.log('typeof fullName:', typeof fullName);
+
+var isCool = true;
+console.log('I am cool:', isCool);
+console.log('typeof isCool:', typeof isCool);
+
+var totalPets = 1;
+console.log('total pets:', totalPets);
+console.log('typeof totalPets:', typeof totalPets);
+
+var unknown = null;
+console.log('unknown:', unknown);
+console.log('typeof unknown:', typeof unknown);
+
+var never =
```

Comment:
this is not correct, you can't use an assignment operator if you are not assigning a value. you want `var never;`